hack/missing-ta-tasks.sh

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+shopt -s globstar
+
+git_root=$(git rev-parse --show-toplevel)
+
+tmp_files=()
+trap 'rm "${tmp_files[@]}" > /dev/null 2>&1' EXIT
+
+# names of workspaces that will not be mounted via PVC, as inline array
+non_pvc_workspaces='["ssh-directory", "basic-auth", "gitops-auth", "git-basic-auth", "netrc"]'
```

Comment:
Not sure if it's worth it, but it would be nice if this list was generated from https://github.com/konflux-ci/build-definitions/blob/7fe56f111b0ea0621efa1e3e530003d547091e28/policies/all-tasks.yaml#L10-L14.

Reply:
I think this is a good idea, incorporated in [`324b27e` (#1523)](https://github.com/konflux-ci/build-definitions/pull/1523/commits/324b27e6d1c737730d051efa1c5763f0da47a670)